src/ssl.c

```diff
@@ -26150,14 +26150,27 @@ int wolfSSL_BIO_printf(WOLFSSL_BIO* bio, const char* format, ...)
     #if defined(OPENSSL_EXTRA) && !defined(_WIN32)
         case WOLFSSL_BIO_SSL:
             {
+                int count;
                 char* pt = NULL;
-                ret = vasprintf(&pt, format, args);
-                if (ret > 0 && pt != NULL) {
-                    wolfSSL_BIO_write(bio, pt, ret);
-                }
-                if (pt != NULL) {
-                    free(pt);
+                va_list copy;
+
+                va_copy(copy, args);
```

Comment:
Does va_start need to be called before va_copy?

Reply:
No, it is not needed.